sheetstorm/urls.py

```diff
@@ -14,7 +14,10 @@
 from django.contrib import admin
 from django.urls import include
 
+from sheetstorm import views
```

Comment:
```suggestion
from . import views
```
I guess it might by more django-way ;)

Reply:
Sure, changed :)